content/automate/ManageIQ/Service/Generic/StateMachines/Utils.class/__methods__/util_object.rb

```diff
@@ -0,0 +1,63 @@
+#
+# Description: Service utils methods
+#
+module ManageIQ
+  module Automate
+    module Service
+      module Generic
+        module StateMachines
+          module Utils
+            class UtilObject
+              def self.dump_root(handle = $evm)
+                handle.log(:info, "Root:<$evm.root> Attributes - Begin")
+                handle.root.attributes.sort.each { |k, v| handle.log(:info, "  Attribute - #{k}: #{v}") }
+                handle.log(:info, "Root:<$evm.root> Attributes - End")
+                handle.log(:info, "")
+              end
+
+              def self.update_task(message, status = nil, handle = $evm)
+                task_name = case handle.root["request"]
+                            when "service_reconfigure"
+                              "service_reconfigure_task"
+                            when "service_retire"
+                              "service_retire_task"
+                            else
+                              "service_template_provision_task"
+                            end
+                handle.root[task_name].try do |task|
+                  task.miq_request.user_message = message
+                  task.message = status unless status.nil?
```

Comment:
Really minor but we tend to prefer the "positive" `if status` over the "negative" `unless status.nil?`